src/Controller/TwitterController.php

```diff
@@ -145,10 +145,10 @@ public function logUserIn(Request $request, PDO $db)
                 throw new Exception('Could not retrieve user-informations from Twitter', Http::FORBIDDEN, $e);
             }
 
-            if ($res->getStatusCode() == Http::OK) {
+            if ($res->getStatusCode() === Http::OK) {
                 $result = $this->oauthModel->createUserFromTwitterUsername(
                     $clientId,
-                    json_decode($res->getBody()->getContents(), JSON_OBJECT_AS_ARRAY)
+                    json_decode($res->getBody()->getContents(), null)
```

Comment:
```suggestion
                    json_decode($res->getBody()->getContents(), true)
```

We use `$result` as an array in L164